sopel/modules/url.py

```diff
@@ -313,7 +319,7 @@ def title_auto(bot: SopelWrapper, trigger: Trigger):
             # Guard against responding to other instances of this bot.
             if message != trigger:
                 bot.say(message)
-        bot.memory["last_seen_url"][trigger.sender] = url
+        bot.memory['last_seen_url'][trigger.sender] = url
```

Comment:
Why the no-op quote change? It's not even consistent with the above rewrite.

Reply:
Huh. Don't know. I may have gotten confused in trying to keep the diff tight? Fixed